File: Tamim/app/src/main/java/com/dma/tamim/Retrofit/IMenuRequest.java
package com.dma.tamim.Retrofit;

import com.dma.tamim.Model.Item;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface IMenuRequest {
    @GET
    Call<List<Item>> getMenuList(@Url String url);
}

File: KotlinSwipeToDelete/app/src/main/java/com/blackbirds/shakil/kotlinswipetodelete/Retrofit/IMenuRequest.kt
package com.blackbirds.shakil.kotlinswipetodelete.Retrofit

import com.blackbirds.shakil.kotlinswipetodelete.Model.Item
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

interface IMenuRequest {
    @GET("json/menu.json")
    fun getMenuList(): Call<MutableList<Item>>
}